Test/civl/inductive-sequentialization/NBuyer.bpl

```diff
@@ -154,6 +154,7 @@ modifies QuoteCH, RemCH, DecCH, contribution;
     RemCH := (lambda i:int :: (lambda r:int :: if i == n && r == price - sum(contribution, 1, n-1) then 1 else 0));
     PAs := MapAddPA(SingletonPA(SellerFinish(0)), SingletonPA(LastBuyer(n)));
     choice := LastBuyer(n);
+    assume RemCH[n][price - sum(contribution, 1, n-1)] > 0;
```

Comment:
The two new assumes are not necessary, if `n-1` is added to the `INV3` pool.

Reply:
I added n-1 to INV3 and also attached INV3 to the lambda bound variables.   The proof succeeds on my machine.